### DISCUSSION TITLE
StarStruck Achievement

### DISCUSSION THREAD

**Discussion by RelxOff** [community/community#50566] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Why don't I get the Starstruck Achievement? Although I am one of the developers in the project, I am not the repository owner.

**Comment by radar-blip** [comment, marked as answer]
You have to be the creator (and owner) of the repository.

See here for more info: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)
